docs/data/material/components/transfer-list/transfer-list.md

```diff
@@ -3,6 +3,7 @@ productId: material-ui
 title: Transfer list React component
 components: List, ListItem, Checkbox, Switch
 githubLabel: 'scope: transfer list'
+disableMarkdownView: true
```

Comment:
Need to ignore this because technically, it's not a React component within the `src`.